deployment/scripts/run_dcl_node

```diff
@@ -0,0 +1,459 @@
+    fi
+}
+
+    
+function run_node {
+    set -eu
+    set -o pipefail
+
+    local _debug="${DEBUG:-}"
+
+    if [[ -z "$_debug" ]]; then
+        # Ubuntu only
+        grep -q -i -e 'ubuntu' /etc/*-release && {
+            # Enable the service
+            sudo systemctl enable dcld
+            # Start node
+            sudo systemctl start dcld
+        } || { echo "Warning: Not an Ubuntu system. Skipping service start..."; return 0; }
+
+        echo "Node has been stared as a service."
+
+        echo -e "\tUse 'systemctl status dcld' to get the node service status."
+        echo "Use 'journalctl -u dcld.service -f' to see node logs."
+    else
+        dcld start &
+        echo "Node has been stared in the backgroud."
+    fi
+}
+
+
+function verify_node {
+    set -eu
+    set -o pipefail
+
+    local _node_ip="${1:-0.0.0.0:26657}"
+
+    echo "Checking a node's 'sync_info.latest_block_height' value is not increasing..."
+
+    echo "sync_info.latest_block_height Try 1"
+    check1="$(dclcli status --node "$_node_ip" | grep latest_block_height)"
+
+    echo "sync_info.latest_block_height Try 2"
+    sleep 6
+    check2="$(dclcli status --node "$_node_ip" | grep latest_block_height)"
+
+    if [[ "$check1" == "$check2" ]]; then
+        echo "Error: sync_info.latest_block_height value is not increasing..."
+        exit 1
+    fi
+
+    # TODO improve: makes sense only for initial genesis node
+    echo "Checking the node is participating in consensus..."
+    dclcli tendermint-validator-set | grep -q address
+
+    # TODO check: catching_up is false
+
+    echo "Passed"
+}
+
+
+function add_account {
+    set -eu
+    set -o pipefail
+
+    local _acc_address="$1"
+    local _acc_pubkey="$2"
+    local _acc_roles="$3"
+    local _from="$4"
+
+    dclcli tx auth propose-add-account --address="$_acc_address" --pubkey="$_acc_pubkey" \
+        --roles="$_acc_roles" --from="$_from"
+
+    echo "Account added"
+    echo "To approve run: dclcli tx auth approve-add-account --address=<string> --from=<account>"
+}
+
+
+function approve_account {
+    set -eu
+    set -o pipefail
+
+    local _acc_address="$1"
+    local _from="$2"
+
+    dclcli tx auth approve-add-account --address="$_acc_address" --from="$_from"
+
+    echo "Account approved by $_from"
+}
+
+# =======================================================
+
+parse_args "$@"
+
+if [[ "$verbosity" -ge 2 ]]; then
+    set -x
+fi
+
+if [[ "$verbosity" -ge 1 ]]; then
+    echo ""
+    #parsed_args=""
+    #parsed_args+="\t CHAIN_ID=$output_dir\n\toutput_type=$output_type"
+    #parsed_args+="\n\tin_docker=$in_docker\n\tinput_dir=$input_dir"
+    #parsed_args+="\n\tfpm_tool=$fpm_tool\n\tverbosity=$verbosity"
+    #parsed_args+="\n\trelease=$pkg_version"
+
+    #echo -e "Parsed arguments:\n$parsed_args"
+fi
+
+echo "Configuring CLI"
+
+if [[ "$NODE_TYPE" == "observer" ]]; then
+    # FIXME is it right
+    config_cli "$CHAIN_ID"
```

Comment:
We need a working local CLI in case of Observer as well, don't we? For example, to check the node status.
So, we need to specify peer there (or use localhost).

Reply:
kept that as default setting, cli should communicate with  the local node